multiply_ui/server/controller.py

```diff
@@ -4,3 +4,13 @@
 def get_parameters(ctx):
     json_text = pkg_resources.resource_string("multiply_ui", "server/resources/processing-parameters.json")
     return json.loads(json_text)
+
+def get_request(ctx, parameters):
+        time_range = parameters["request"]["timeRange"]
+        region_wkt = parameters["request"]["regionWKT"]
```

Comment:
I understand that `regionWKT` is super-generic and therefore useful. However, we currently provide only a BBox and this is what can be edited in the GUI. To edit a WKT in a BBox GUI we had to parse the BBox from WKT first which is extra dev work. I'd suggest staying with `regionBBox` (or just `bbox` --> see OpenSearch) using the format `lon-min>,lat-min,lon-max,lat-max`

Reply:
Done with next commit.